pandas/tests/frame/test_repr_info.py

```diff
@@ -26,14 +26,16 @@
 
 
 class TestDataFrameReprInfoEtc:
-    def test_repr_bytes_61_lines(self):
+    def test_repr_bytes_61_lines(self, using_array_manager):
         # GH#12857
         lets = list("ACDEFGHIJKLMNOP")
         slen = 50
         nseqs = 1000
         words = [[np.random.choice(lets) for x in range(slen)] for _ in range(nseqs)]
         df = DataFrame(words).astype("U1")
-        assert (df.dtypes == object).all()
+        # TODO(Arraymanager) astype("U1") actually gives this dtype instead of object
```

Comment:
im pretty sure we dont want this behavior?  so should xfail for now?

Reply:
This test is not actually testing `astype`, but rather `repr`, so I would prefer to run the rest of the test (which actually passes) without having this line error. 

Now, I assume we should have an astype-specific test about this as well, that could be xfailed.

Reply:
I don't directly see a test about it in `frame/methods/test_astype.py`, so I will add a test for that there. It's not very clear, though, what the expected behaviour should be.

Reply:
could move the repr calls up and do the dtype assertion at the end